frontend/src/scenes/session-recordings/player/inspector/components/PlayerInspectorListItem.tsx

```diff
@@ -59,7 +60,99 @@ const typeToIconAndDescription = {
         tooltip: 'A user commented on this timestamp in the recording',
     },
 }
-const PLAYER_INSPECTOR_LIST_ITEM_MARGIN = 4
+const PLAYER_INSPECTOR_LIST_ITEM_MARGIN = 1
+
+function ItemTimeDisplay({ item }: { item: InspectorListItem }): JSX.Element {
+    const { timestampFormat } = useValues(playerSettingsLogic)
+    const { logicProps } = useValues(sessionRecordingPlayerLogic)
+    const { durationMs } = useValues(playerInspectorLogic(logicProps))
+
+    const fixedUnits = durationMs / 1000 > 3600 ? 3 : 2
+
+    return (
+        <span className="px-2 py-1 text-xs">
+            {timestampFormat != TimestampFormat.Relative ? (
+                (timestampFormat === TimestampFormat.UTC ? item.timestamp.tz('UTC') : item.timestamp).format(
+                    'DD, MMM HH:mm:ss'
+                )
+            ) : (
+                <>
+                    {item.timeInRecording < 0 ? (
+                        <Tooltip
+                            title="This event occured before the recording started, likely as the page was loading."
+                            placement="left"
+                        >
+                            <span className="text-muted">load</span>
+                        </Tooltip>
+                    ) : (
+                        colonDelimitedDuration(item.timeInRecording / 1000, fixedUnits)
+                    )}
+                </>
+            )}
+        </span>
+    )
+}
+
+function RowItemTitle({
```

Comment:
split the items into a title and a detail component
can get more clever here since there is some repetition in container classes and the like but wanted to keep things as small as possible for this PR